modules/nw-ovn-k-baseline-adminnetwork-policy.adoc

```diff
@@ -0,0 +1,137 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn-k-network-policy.adoc
+
+:_mod-docs-content-type: <Concept>
+[id="BaselineAdminNetworkPolicy"_{context}"]
+= BaselineAdminNetworkPolicy
+
+`BaselineAdminNetworkPolicy` (BANP) is a cluster-scoped custom resource definitions (CRDs). As an {product-title} administrator, you can use `BaselineAdminNetworkPolicy` to setup and enforce optional baseline network policies that are overridable by users if need be. Rule actions for BANP are `allow` or `deny`.
```

Comment:
```suggestion
`BaselineAdminNetworkPolicy` (BANP) is a cluster-scoped custom resource definition (CRD) shipped and maintained by blah -> same as suggested for ANP. As an {product-title} administrator, you can use `BaselineAdminNetworkPolicy` to setup and enforce optional baseline network policies that are overridable by users using NetworkPolicies if need be.
```

Rule actions for BANP are `allow` or `deny` -> we probably want to put this somewhere else like near to the BANP example action line? (I think you already have...)